src/elements/content-uploader/DroppableContent.js.flow

```diff
@@ -67,7 +65,7 @@ const DroppableContent = makeDroppable(dropDefinition)(({
 
     return (
         <div className="bcu-droppable-content">
-            <ItemList items={items} onClick={onClick} view={view} />
+            <ItemList items={items} onClick={onClick} />
```

Comment:
view is not used anywhere in ItemList component

Reply:
Are we using anys for any particular reason? Is there a plan to add more specificity down the road?

Reply:
great catch, they are automatically updated to `any` type after running the script to migrate js to tsx file. I just updated it with more specific types as what is being used in the tsx file.